ex03.ex

```diff
@@ -101,7 +119,25 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([head1 | tail1], [head2 | tail2]) when head1 == head2 do
```

Comment:
No deduction. 

You thought through all the corner cases, which was good.

It could be simpler though:

~~~ elixir
  def list_equal([], []),                              do: true
  def list_equal([ h | t1 ], [h | t2]),                do: list_equal(t1, t2)
  def list_equal(a, b) when is_list(a) and is_list(b), do: false
~~~